packages/base-types/src/node/visual.ts

```diff
@@ -62,6 +62,9 @@ export interface ImageStepData extends BaseStepData {
   frameType?: FrameType;
   dimensions: Nullable<Dimensions>;
   canvasVisibility: CanvasVisibility;
+  options: {
```

Comment:
I think it should be optional
```suggestion
  options?: {
```
and then `loop` can be required inside, reason being all our existing steps in the diagrams don't have the `options` field.